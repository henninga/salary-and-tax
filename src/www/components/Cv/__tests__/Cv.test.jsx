import mockComponent from 'identity-obj-proxy';
import React from 'react';
import renderer from 'react-test-renderer';
import testData from '../../../../__data__/cv.json'
import { CvComponent } from '../index';


jest.mock('../../PersonalDetails', () => mockComponent);
jest.mock('../../Ui', () => mockComponent);

describe('CvComponent', () => {
    it('should render', () => {
        const tree = renderer.create(<CvComponent data={testData} isFetching={false} isLoaded />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('should not display when loading', () => {
        const tree = renderer.create(<CvComponent data={{}} isFetching isLoaded={false} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
