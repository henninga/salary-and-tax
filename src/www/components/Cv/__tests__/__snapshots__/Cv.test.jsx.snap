// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CvComponent should not display when loading 1`] = `<Loading />`;

exports[`CvComponent should render 1`] = `<PersonalDetails />`;
